src/api.js

```diff
@@ -0,0 +1,61 @@
+/* eslint no-console: 0 */
+import axios from 'axios';
+import { route } from 'preact-router';
+
+const HOST = 'localhost:3001';
+// const HOST = 'el-parking-api-staging.herokuapp.com';
```

Comment:
redundant comment. Extract Host to envs and set for development `localhost:3001` and for `production el-parking-api-staging.herokuapp.com`